packages/rs-dpp/src/data_trigger/dashpay_data_triggers/mod.rs

```diff
@@ -41,13 +42,11 @@ where
 
     let core_height_created_at = data.get_i64(PROPERTY_CORE_HEIGHT_CREATED_AT)?;
 
-    let latest_block_header: JsonValue = context
+    let core_chain_locked_height = context
         .state_repository
-        .fetch_latest_platform_block_header()
-        .await?;
-
-    let core_chain_locked_height =
-        latest_block_header.get_i64(PROPERTY_CORE_CHAIN_LOCKED_HEIGHT)?;
+        .fetch_latest_platform_core_chain_locked_height()
```

Comment:
@antouhou I assume `fetch_latest_platform_block_header` is misused here. Replaced with `fetch_latest_platform_core_chain_locked_height`

Reply:
Yep, that seems to be the case